src/Parser/DeathKnight/Frost/Modules/Features/WastedRimeProcs.js

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import Analyzer from 'Parser/Core/Analyzer';
+import SPELLS from 'common/SPELLS';
+import SpellIcon from 'common/SpellIcon';
+import SpellLink from 'common/SpellLink';
+import AbilityTracker from 'Parser/Core/Modules/AbilityTracker';
+import StatisticBox, { STATISTIC_ORDER } from 'Main/StatisticBox';
+import Combatants from 'Parser/Core/Modules/Combatants';
+
+class WastedRimeProcs extends Analyzer {
```

Comment:
see comments on the previous class, it mostly applies here too

Reply:
Also edited according to other suggestions.